packages/manager/modules/pci/src/projects/projects.component.js

```diff
@@ -3,8 +3,9 @@ import template from './projects.html';
 
 export default {
   bindings: {
+    billingUrl: '<',
```

Comment:
Same comment, should it be string binding '@'?

Reply:
This is done